### PR TITLE
Add vexctl package

### PR DESCRIPTION
**Pull request by jspeed-meyers**
This PR packages `vexctl`, a Go tool for creating, transforming, and attesting VEX metadata. It is part of the OpenVEX project.

#### For new package PRs only
- [x] REQUIRED - The package is available under an OSI-approved or FSF-approved license
- [x] REQUIRED - The version of the package is still receiving security updates
- [ ] This PR links to the upstream project's support policy (e.g. `endoflife.date`) (no support policy present)

### CODE REVIEW

File: vexctl.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+package:
+  name: vexctl
+  version: 0.2.3
+  epoch: 0
+  description: A tool to create, transform and attest VEX metadata
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - go
```
Comment (rawlingsj) [inline vexctl.yaml:10]:
```suggestion
```
We can remove the runtime dependency on go

File: vexctl.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - go
```
Comment (rawlingsj) [inline vexctl.yaml:18]:
```suggestion
```
This is automagically added by the [go/install](https://github.com/chainguard-dev/melange/blob/a4590d130eb51801d2819183e5cdc19fbcd079db/pkg/build/pipelines/go/install.yaml#L5) pipeline so can be removed

**Review by rawlingsj — CHANGES_REQUESTED** [review]
looks great! just two minor comments

File: vexctl.yaml (hunk 1 above)
Comment (imjasonh) [inline vexctl.yaml:10]: Just checking: does this actually need go at runtime? Most Go apps don't need Go as a runtime dep, unless maybe they shell out to `go build` or something (like `ko` does).

This is as much an admission of my inexperience about vexctl as anything else.

Comment (jspeed-meyers) [reply vexctl.yaml:10]: Will remove. Cooking up a commit! TY

File: vexctl.yaml (hunk 2 above)
Comment (jspeed-meyers) [reply vexctl.yaml:18]: Ohhh, too cool. Will remove.

Comment (jspeed-meyers) [reply vexctl.yaml]: See commit b17865a

**Comment by jspeed-meyers** [comment]
Thank you, @rawlingsj and @imjasonh.

**Review by rawlingsj — APPROVED** [review]

**Pull request merged**